code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1633,6 +1629,18 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 		H.receiving_cpr = FALSE
 		to_chat(src, "<span class='danger'>You need to stay still while performing CPR!</span>")
 
+/mob/living/carbon/human/proc/get_cpr_mod(mob/living/carbon/human/H)
+	if((head && (head.flags_cover & HEADCOVERSMOUTH)) || (wear_mask && (wear_mask.flags_cover & MASKCOVERSMOUTH) && !wear_mask.mask_adjusted))
+		return CPR_CHEST_COMPRESSION_ONLY
+	if((H.head && (H.head.flags_cover & HEADCOVERSMOUTH)) || (H.wear_mask && (H.wear_mask.flags_cover & MASKCOVERSMOUTH) && !H.wear_mask.mask_adjusted))
+		return CPR_CHEST_COMPRESSION_ONLY
+	if(!H.check_has_mouth() || !check_has_mouth())
+		return CPR_CHEST_COMPRESSION_ONLY
```

Comment:
This is a bit unpleasant to read, I would shuffle things around and use `?.` to make things a bit cleaner:
```suggestion
/mob/living/carbon/human/proc/get_cpr_mod(mob/living/carbon/human/H)
	if(!H.check_has_mouth() || !check_has_mouth())
		return CPR_CHEST_COMPRESSION_ONLY
	if(head?.flags_cover & HEADCOVERSMOUTH || (wear_mask && (wear_mask.flags_cover & MASKCOVERSMOUTH) && !wear_mask.mask_adjusted))
		return CPR_CHEST_COMPRESSION_ONLY
	if(H.head?.flags_cover & HEADCOVERSMOUTH || (H.wear_mask && (H.wear_mask.flags_cover & MASKCOVERSMOUTH) && !H.wear_mask.mask_adjusted))
		return CPR_CHEST_COMPRESSION_ONLY
```

One thing you could also do is add an argument to `check_has_mouth()` (or make a new proc) that returns whether the mob's mouth not only exists, but is also non-covered by something.